frontend/ui/widget/configdialog.lua

```diff
@@ -577,15 +577,15 @@ function ConfigOption:init()
                                 self.options[c].more_options_param.show_true_value_func = self.options[c].show_true_value_func
                             end
                             Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_MORE)
-                            local default_value
+                            local default_value_local
```

Comment:
`local default_value_local` sounds odd :)
May be it should be `current_default_value(s)` ?